addons/sale/controllers/portal.py

```diff
@@ -144,8 +144,7 @@ def portal_order_page(
                 author = order_sudo.partner_id if request.env.user._is_public() else request.env.user.partner_id
                 msg = _('Quotation viewed by customer %s', author.name)
                 del context
-                order_sudo.message_post(
-                    author_id=author.id,
+                order_sudo.with_user(SUPERUSER_ID).message_post(
```

Comment:
This is not a 18 issue, so not a FIX, the behavior was kept the same as before, it's the same behavior in 17.0.
We tried to keep the same behavior than before on purpose, and only disable the notifications for those that didn't want it.

It can be considered as a master improvement, but no need for 18.

Reply:
Got it!